src/bionics.cpp

```diff
@@ -944,7 +944,6 @@ bool player::activate_bionic(int b, bool eff_only)
     } else if(bio.id == "bio_lockpick") {
         tmp_item = item( "pseuso_bio_picklock", 0 );
         if( invoke_item( &tmp_item ) == 0 ) {
-            charge_power(bionics["bio_lockpick"].power_activate);
```

Comment:
This line actually _charges_ the batteries (charge of +25 PU, consumption is done in another line);